app/controllers/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
```

Comment:
Habt ihr mal ausprobiert, ob man sich noch anmelden kann?
Das Erstellen des UserControllers macht die bisherigen routen wie z.B. show kaputt.

Reply:
wir können eine leere show methode auf alle fälle in den controller tun aber ich hab das gefühl dass ihr das ja auch gemacht haben müsstet? oder war das nicht was ihr gefixt hattet noch?

Reply:
Ja, für #22 brauchten wir keinen Controller und haben ihn gelöscht.
Für die #26 haben wir dann einen ContactsController erstellt.
Man darf nur keinen UserController nehmen.

Reply:
wir lassen den UsersController, weil es für den ContactsController kein Modell gibt und damit ist die semantik eines ContactsController schwieriger... außerdem haben andere Gruppen auch UsersController... früher oder später haben wir den im Code... das Problem ist nicht der Controller gewesen sondern dass wir die Routen auch hinzugefügt haben